components/dashboards-web-component/src/auth/Login.jsx

```diff
@@ -216,6 +216,7 @@ export default class Login extends Component {
                         onSubmit={this.authenticate}
                     >
                         <TextField
+                            id={'txt-username'}
```

Comment:
Shall we put this as `id="txt-username"` as no executable code within? Do the same for the rest.

Reply:
Did the suggested changes